### PR TITLE
#3073 -  Student application tracker card for disbursement feedback error

### PR DESCRIPTION
**Pull request by dheepak-aot**
# Student application tracker card for disbursement feedback error

## Tracker card and Progress bar
- [x] Display tracker card and progress bar status when the application current assessment has one or more feedback errors that block funding.
image

## E2E Tests
- `getApplicationProgressDetails` API
image

- `getCompletedApplicationDetails` API

image

### CODE REVIEW

File: sources/packages/backend/apps/api/src/route-controllers/application/application.students.controller.ts [hunk 1]
```diff
@@ -694,6 +700,7 @@ export class ApplicationStudentsController extends BaseController {
       scholasticStandingChangeType: scholasticStandingChange?.changeType,
       applicationOfferingChangeRequestStatus,
       assessmentTriggerType,
+      hasFeedbackError,
```
Comment (andrewsignori-aot) [inline sources/packages/backend/apps/api/src/route-controllers/application/application.students.controller.ts:703]: Should it also contain the blocking part in the name (e.g. `hasBlockingFeedbackError`)?

File: sources/packages/backend/apps/api/src/route-controllers/application/_tests_/application.students.controller.getCompletedApplicationDetails.e2e-spec.ts [hunk 2]
```diff
@@ -313,9 +319,169 @@ describe("ApplicationStudentsController(e2e)-getCompletedApplicationDetails", ()
...
+        {
+          applicationStatus: ApplicationStatus.Completed,
+          offeringIntensity: OfferingIntensity.partTime,
+          firstDisbursementInitialValues: {
+            coeStatus: COEStatus.completed,
+            disbursementScheduleStatus: DisbursementScheduleStatus.Sent,
+          },
+        },
+      );
+      const [firstDisbursement] =
```
Comment (andrewsignori-aot) [inline sources/packages/backend/apps/api/src/route-controllers/application/_tests_/application.students.controller.getCompletedApplicationDetails.e2e-spec.ts:396]: As a suggestion, we can have at least one between all new E2Es checking for the second disbursement with the feedback errors.

**Review by andrewsignori-aot — COMMENTED** [review]
Nice work, only minor comments/suggestions, please take a look.

File: sources/packages/backend/apps/api/src/route-controllers/application/application.students.controller.ts (hunk 1 above)
Comment (dheepak-aot) [reply sources/packages/backend/apps/api/src/route-controllers/application/application.students.controller.ts:703]: Yes I was juggling back and forth with that thought and went this way. I can change it.

what about `hasBlockFundingFeedbackError`?

**Review by andrewsignori-aot — APPROVED** [review]
Thanks for making the changes, looks good 👍

**Pull request merged**